CriticalMass/de.lproj/Localizable.strings

```diff
@@ -3,7 +3,7 @@
 "chat.noChatActivity" = "Niemand chattet derzeit...\nWarum fängt du nicht an?";
 
 /* No comment provided by engineer. */
-"chat.placeholder" = "Nachricht ...";
+"chat.placeholder" = "Nachricht Schreiben";
```

Comment:
Think this needs to be `Nachtricht schreiben`

Reply:
```suggestion
"chat.placeholder" = "Nachricht schreiben";
```

It should just be a small `s` :)